README.md

```diff
@@ -222,7 +222,7 @@ go test -v ./...
 
 1. Bump the version number in `main.go` [here](https://github.com/taskcluster/generic-worker/blob/d1e48692122dd3e295defda1e61acc8509ad7e23/main.go#L58).
 2. Commit the change, e.g. `git add main.go; git commit -m "Bumped version number"`.
-3. Tag the repo, e.g. `git tag v2.0.0`
+3. Tag the repo, e.g. `git tag v2.1.0`
```

Comment:
I do it because I want people to _feel_ like docs are up-to-date. If they read the example and it is about an ancient release, it will feel like the docs have not been updated since that old release. Making it current just gives the user the feeling that this doc is maintained and relevant, IMHO. It's just a minor detail to hopefully improve trust for the user.